common/openapi.yml

```diff
@@ -106,6 +106,202 @@ paths:
           application/json:
             schema:
               type: string
+            examples:
+              jsonObject:
+                summary: A sample object
```

Comment:
I've looked into this more and I think we should be able to get `externalValue` to work. The problem is that the `index.html` that ships with swagger doesn't support it yet. See https://github.com/swagger-api/swagger-ui/issues/5433

For now, you should look at the example HTML solutions on that GitHub issue and add one to our copy of `index.html` (which is in `service/src/main/resources/templates`).